docs/_includes/toggle-button-group.html

```diff
@@ -0,0 +1,1349 @@
+                        <span class="toggle-button__content">
+                            <span class="toggle-button__title">8.5</span>
+                        </span>
+                    </button>
+                </li>
+            </ul>
+        </div>
+    </div>
+
+{% highlight html %}
+<ul aria-label="Select Shoe Size" class="toggle-button-group" data-columns="2">
+    <li>
+        <button type="button" class="toggle-button" aria-pressed="false">
+            <span class="toggle-button__content">
+                <span class="toggle-button__title">8</span>
+            </span>
+        </button>
+    </li>
+    <li>
+        <button type="button" class="toggle-button" aria-pressed="false">
+            <span class="toggle-button__content">
+                <span class="toggle-button__title">8.5</span>
+            </span>
+        </button>
+    </li>
+</ul>
+{% endhighlight %}
+
+    <h3>Prescriptive Responsive Layouts Using Preferred Minimum Columns</h3>
+    
+    <p>You can provide the preferred minimum number of columns to display per row by adding the attribute <span class="highlight">data-columns="x"</span> to the Toggle Button Group where <span class="highlight">x</span> is how many buttons you'd prefer to have display per row. Keep in mind, however, that this doesn't guarantee you will end up with the number of columns specified as the preference. The columns will responsively shrink and stretch to the limits of min/max widths of the specific type of button in use and attempt to fit the buttons nicely within the container.</p>
+
+    <section class="section-notice section-notice--information" role="region" aria-label="Information">
+        <div class="section-notice__header">
+            <svg class="icon icon--information-filled-16" focusable="false" height="16" width="16" role="img" aria-label="Information">
+                {% include symbol.html name="information-filled-16" %}
+            </svg>
+        </div>
+        <div class="section-notice__main">
+            <h3 class="section-notice__title">Preferred Minimum Columns</h3>
+            <p>When there is more than enough space after the max button widths have been reached, the layout will try to fit more buttons.</p>
+
+            <p>When there's not enough space after the minimum button widths have been reached and, as a result, the number of buttons per row no longer fits within the container, the buttons will start collapsing/wrapping to the next row automatically.</p>
+        </div>
+    </section>
+
+    <p>Currently, Toggle Button Group is limited to 6 columns of buttons inside the group.</p>
+
+    <p>You will frequently see examples with light yellow backgrounds applied to containers. To clarify, for real-world application, your own container where you drop in Toggle Button Group should not look this way. Those are styled differently here to easily show the current container element boundaries.</p>
```

Comment:
Maybe this could be in a section notice instead, since it applies to all future toggle button groups.